src/sentry/constants.py

```diff
@@ -415,7 +422,7 @@ class ObjectStatus:
     DISABLED = 1
 
     @classmethod
-    def as_choices(cls):
+    def as_choices(cls) -> Sequence[Tuple[int, str]]:
```

Comment:
You could define a `NewType[int]` for this

Reply:
I think ideally this would be an enum and the return type would be the `Tuple[ObjecStatus, str]` (or really a mapping, tuple is kind of weird too).  But for now I was trying to not make any changes to the actual code of something so shared as this.  My hope is that types become stricter as they are more widely adopted.

Reply:
Right, NewType does not require you to make changes to runtime types though, that's the point. It's just to tell mypy that it should treat this int as separate type

Reply:
fair enough, i see your point.  how does it treat using that returned in then?  will it refuse me using it as an int somewhere?  i guess that could be useful

Reply:
yeah and vice-versa, forces you to use the constants